### PR TITLE
docs: add Getting Started section

### PR DESCRIPTION
**Pull request by sanderblue**

### CODE REVIEW

File: website/docs/guides/getting_started.html.markdown [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+---
+
+# Getting Started with the New Relic Provider
+
+## Before You Begin
+
+* This guide assumes you already have a New Relic agent deployed. If you don't have New Relic integrated yet, check out [New Relic's introduction documentation](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/introduction-new-relic) to get started there, then head back over here to get started with the New Relic Terraform Provider.
+* [Install Terraform](https://www.terraform.io/intro/getting-started/install.html)
+and read the Terraform getting started guide that follows. This guide will
+assume a basic understanding of Terraform.
```
Comment (ctrombley) [inline website/docs/guides/getting_started.html.markdown]: nit: unnecessary line breaks?

File: website/docs/guides/getting_started.html.markdown [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+* Locate your API key by following [New Relic's Admin API key docs](https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#admin).
```
Comment (ctrombley) [inline website/docs/guides/getting_started.html.markdown]: Let's use "Admin's API key" for clarity.

File: website/docs/guides/getting_started.html.markdown (hunk 1 above)
Comment (sanderblue) [reply website/docs/guides/getting_started.html.markdown]: Not sure which line break you're referring to here, but Terraform's markdown flavor has several subtle differences with how it parses the markdown - line breaks being a part of that.

File: website/docs/guides/getting_started.html.markdown (hunk 2 above)
Comment (sanderblue) [reply website/docs/guides/getting_started.html.markdown]: Good call. Updated.

**Pull request merged**